include/aikido/constraint/FrameTestable.hpp

```diff
@@ -29,7 +29,13 @@ class FrameTestable : public Testable
   /// \param _state a MetaskeletonState to set the configuration of this
   ///        constraint's metaskeketon. This state's StateSpace should match
   ///        StateSpace returend by getStateSpace().
```

Comment:
Document the `TestableOutcome` argument. I suggest making it simply return whatever `TestableOutcome` is returned by the `_poseConstraint` passed to this class' constructor.